extensions/emmet/src/test/wrapWithAbbreviation.test.ts

```diff
@@ -356,6 +356,54 @@ suite('Tests for Wrap with Abbreviations', () => {
 			});
 		});
 	});
+
```

Comment:
I have added another test in the master branch for multi-line selection which will fail with the changes in this PR. See https://github.com/Microsoft/vscode/commit/6f139c79ad8a00ff7dd0f0da0576a5296ac95eb4

Reply:
This test passed as well with the last fix.